### PR TITLE
Cleanup CI

### PR DESCRIPTION
**Pull request by bquorning**

### CODE REVIEW

File: gemfiles/activerecord_5.2.gemfile.lock [hunk 1]
```diff
@@ -61,7 +60,6 @@ DEPENDENCIES
   rake
   rspec (~> 3.9)
   sqlite3
-  wwtd
 
 BUNDLED WITH
    1.17.3
```
Comment (grosser) [inline gemfiles/activerecord_5.2.gemfile.lock:65]: afaik this will crash and is why I added the matching_bundler

Comment (bquorning) [reply gemfiles/activerecord_5.2.gemfile.lock:65]: In my experience, ruby/setup-ruby handles installing the correct bundler version.

**Comment by bquorning** [comment]
I am confused by those failing tests – it looks like the database already exists?

**Comment by bquorning** [comment]
> it looks like the database already exists?

🤦🏼 I had added the database file to the repository. It's now in the .gitignore file.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/file::memory:?cache=shared
```
Comment (grosser) [inline .gitignore:1]: this is not correct, needs to be a real file ... already saw this in another repo
it's supposed to be in-memory but that broke

Comment (bquorning) [reply .gitignore:1]: Yeah, it's weird. But it created a file on my machine 🤷🏼

Comment (grosser) [reply .gitignore:1]: see https://github.com/grosser/ar_multi_threaded_transactional_tests/blob/master/test/database.rb

File: gemfiles/activerecord_5.2.gemfile.lock (hunk 1 above)
Comment (grosser) [reply gemfiles/activerecord_5.2.gemfile.lock]: if it works then that's great :D

File: .gitignore (hunk 2 above)
Comment (bquorning) [reply .gitignore]: I have removed the .gitignore file. But I am not sure how to fix sqlite. Is it sqlite or the sqlite adapter that is broken?

**Comment by bquorning** [comment]
I have removed Ruby 3.0 from the build for now. Will open a new pull request specifically for 3.0 support.

File: Rakefile [hunk 3]
```diff
@@ -1,14 +1,13 @@
 require "bundler/setup"
 require "bundler/gem_tasks"
 require "bump/tasks"
-require "wwtd/tasks"
 require "appraisal"
```
Comment (bquorning) [inline Rakefile:4]: Do you still use Appraisal, @grosser, or do you edit gemfiles manually? (I've edited them manually in this PR)

**Comment by grosser** [comment]
I usually edit them manually, so maybe just remove appraisal

**Comment by grosser** [comment]
sqlite has a bug were the old shared memory no longer works, so need to use
a file

**Comment by bquorning** [comment]
> maybe just remove appraisal

Sounds like a separate PR.

**Comment by grosser** [comment]
sure

**Pull request merged**